packages/localize/test/LocalizeManager.test.js

```diff
@@ -405,5 +405,14 @@ describe('LocalizeManager', () => {
       manager.addData('en-GB', 'overridden-ns', { greeting: 'Hello!' });
       expect(manager.msg(keys)).to.equal('Hello!');
     });
+
+    it('gently handle missing namespace prefix', () => {
```

Comment:
`it('throws a custom error when namespace prefix is missing', () => ` might be more descriptive / accurate

Reply:
Agree. Let me change it.